src/lib/fcitx/userinterface.cpp

```diff
@@ -10,4 +10,14 @@
 namespace fcitx {
 
 UserInterface::~UserInterface() {}
+
+bool UserInterface::isVirtualKeyboard() {
+    return false;
+}
+
+VkUserInterface::~VkUserInterface() {}
```

Comment:
Use full name instead of "vk".

Reply:
I will replace "vk" with "VirtualKeyboard"